qemu/tests/cfg/balloon_stress.cfg

```diff
@@ -24,6 +24,7 @@
         play_video_cmd = '"%s" "%s" /play /fullscreen'
         time_for_video = 1200
         guest_alias = "Win2008-sp2-32:2k8\x86,Win2008-sp2-64:2k8\amd64,Win2008-r2-64:2k8\amd64,Win2012-64:2k12\amd64,Win2012-64r2:2k12\amd64"
+        ratio = 0.5
```

Comment:
The value is refered to balloon_check.cfg, but this case is about stress test,is it better to set a big value?

Reply:
0.5 should be big enough:
used_size = int(self.get_ballooned_memory() + self.ratio * self.ori_mem)
min_size = max(used_size, min_size)

If guest ori_mem is small such 1024M, min_size is about 900M.
If set ratio too big, min_size sometimes will larger than ori_mem.

ratio value can be set to a bigger value in the future if we find it's not enough.